### ISSUE TITLE
Use apps.openbudgets.eu ; not eis-openbudgets.iais.fraunhofer.de

### ISSUE THREAD

**Issue by pwalsh** [openbudgets/platform#46]
# Description

`eis-openbudgets.iais.fraunhofer.de` is not communicative. I suggest we use the domain `apps.openbudgets.eu` as the entry point to the OpenBudgets.eu apps instead. I've set up a CNAME record for this, so if it is agreed, then there is just one task to do on the server (details below).

# Tasks

- [ ] Update the frontend servers at Fraunhofer to accept `apps.openbudgets.eu` where `eis-openbudgets.iais.fraunhofer.de` is currently used.

@badmotor @larjohn @skarampatakis @jindrichmynarz @jakubklimek agreed, or, any comments?

**Comment by skarampatakis** [comment]
Totally agree.

**Comment by larjohn** [comment]
Yes!

**Comment by pwalsh** [comment]
@liyakun seeing as several agree, I'm assigning to you. You should continue to accept requests on the current domain (Ideally by rewriting them to `apps.openbudgets.eu`), and also accept requests, as per the original issue, to `apps.openbudgets.eu`.

**Comment by liyakun** [comment]
@pwalsh I redirected  `eis-openbudgets.iais.fraunhofer.de` to `apps.openbudgets.eu`.

**Comment by liyakun** [comment]
@pwalsh I don't understand what you mean by 

> and also accept requests, as per the original issue, to apps.openbudgets.eu

**Comment by pwalsh** [comment]
@liyakun not sure how else to explain - this is just basic web routing.

For example, if you have configured correctly, then `http://apps.openbudgets.eu/explorer/` should return the explorer, but it does not.

**Comment by liyakun** [comment]
@pwalsh thanks, understand, it works now, some small problems should still need to be take care of.

**Comment by liyakun** [comment]
@pwalsh I switched back to `eis-openbudgets.iais.fraunhofer.de` as there are some problems with some componets, will check later.

**Comment by liyakun** [comment]
@pwalsh I changed the domain from `eis-openbudgets.iais.fraunhofer.de` to `apps.openbudgets.eu`.

**Comment by jindrichmynarz** [comment]
When requesting <http://apps.openbudgets.eu>, the AJAX call to <http://apps.openbudgets.eu/search/package?size=10000> times out (HTTP 504).

**Comment by liyakun** [comment]
@jindrichmynarz yeah, it happens, as rudolf updates every 30 minutes.

**Comment by liyakun** [comment]
@pwalsh the redirection from `eis-openbudgets.iais.fraunhofer.de` to `http://apps.openbudgets.eu` has been done.

**Comment by larjohn** [comment]
@pwalsh we are having a couple of issues with script bundles being cached by Cloudflare and this makes it harder to deploy rapid changes at this phase of development.

Could you temporarily disable caching for apps.openbudgets.eu in Cloudflare?

**Comment by liyakun** [comment]
@larjohn it seems that the problem has been solved, I will close this issue then.

**Comment by larjohn** [comment]
This seems to be solved because we moved from `/indigo` to `/`, so we actually requested a new file that was not in the cache anyway. 

Eventually the same thing will happen again. Two solutions come in mind. Either @pwalsh (at least temporarily) disables the caching for the domain or @liyakun examines wether we can successfully control cache life using the nginx configuration. 

Regarding the latter, I have seen a couple of configuration settings in our own docker container, so it might be trivial if Cloudflare does take into account the headers of the response to treat cache as defined.
